source/androidx.media3/media3-exoplayer/Additions/AndroidX.Media3.ExoPlayer.IExoPlayer.cs

```diff
@@ -0,0 +1,7263 @@
+using System;
+using System.Collections.Generic;
+using Android.Runtime;
+using Java.Interop;
+
+namespace AndroidX.Media3.ExoPlayer {
```

Comment:
This is a lot of hand-binding to maintain, what issue prevents `generator` from binding this?

Reply:
Not sure. It was long time ago. I think one of the problems was arity with generics.